docker-compose-odoo.yml

```
@@ -96,6 +97,8 @@ services:
       - EIP_FHIR_PASSWORD=${OPENMRS_PASSWORD}
       - EIP_TEST_ORDER_TYPE_UUID=${EIP_TEST_ORDER_TYPE_UUID}
     image: mekomsolutions/eip-client
+    labels:
+      - "app_name=eip"
```

Let's be consistent with the others apps with the formatting:
```suggestion
      app_name: "eip"
```